src/main.jsx

```diff
@@ -1,9 +1,12 @@
-import React from 'react'
-import ReactDOM from 'react-dom/client'
-import App from './App.jsx'
+import React from "react";
+import ReactDOM from "react-dom/client";
+import "../src/assets/css/main.css";
+import "bootstrap/dist/css/bootstrap.min.css";
+import { RouterProvider } from "react-router-dom";
+import { router } from "./routes/Router";
 
```

Comment:
Keeping imports in order and separated properly by spacing can help in improved readability.

import packages

import components

import helper files

import CSS (if any)